### PR TITLE
feat: [CDS-86805]: Serverless V2 - Download Manifest from Harness Store Support for ServerlessAwsLambda Manifest

### PR DESCRIPTION
**Pull request by shivanand-harness**
### Summary

Jira Ticket: https://harness.atlassian.net/browse/CDS-86805

Support harness store in ServerlessV2 manifest

#### Screenshots

Services:

https://github.com/harness/harness-core-ui/assets/148759765/a4ff34c9-d2b4-428a-a290-01c863a55d41

Pipelines: 

https://github.com/harness/harness-core-ui/assets/148759765/4d9761bb-398c-4e92-8cd2-d036479824c0

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
- MessageMetadata: `trigger messagecheck`
- GitLeaks: `trigger gitLeaks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

### CODE REVIEW

**Comment by shivanand-harness** [comment]
trigger messagecheck

**Comment by shivanand-harness** [comment]
trigger messagecheck

File: src/modules/27-platform/filestore/components/FileStoreList/FileStoreList.tsx [hunk 1]
```diff
@@ -149,6 +158,11 @@ export const FileSelectList = (props: ListProps): React.ReactElement => {
     }
   }, [initialValue, name])
 
+  const errorCheck = (field: string): boolean =>
+    ((get(formik?.touched, field) || (formik?.submitCount && formik?.submitCount > 0)) &&
+      get(formik?.errors, field) &&
+      !isPlainObject(get(formik?.errors, field))) as boolean
```
Comment (ishantmehtaharness) [inline src/modules/27-platform/filestore/components/FileStoreList/FileStoreList.tsx:164]: What is the use case of using isPlainObject ?

Comment (ishantmehtaharness) [reply src/modules/27-platform/filestore/components/FileStoreList/FileStoreList.tsx:164]: What if the internal formik implementation changes?

Comment (shivanand-harness) [reply src/modules/27-platform/filestore/components/FileStoreList/FileStoreList.tsx:164]: > What if the internal formik implementation changes?

ExpressionInput component does not handle showing error message scenario so need to handle explicitly

**Review by chetanvekariya-harness — APPROVED** [review]

**Review by rameshx — APPROVED** [review]

**Pull request merged**